src/megachatapi_impl.cpp

```diff
@@ -1408,6 +1408,14 @@ void MegaChatApiImpl::fireOnMessageUpdate(MegaChatMessage *msg)
     delete msg;
 }
 
+void MegaChatApiImpl::fireOnHistoryReloaded(MegaChatRoom *chat)
+{
+    for(set<MegaChatRoomListener *>::iterator it = roomListeners.begin(); it != roomListeners.end() ; it++)
+    {
+        (*it)->onHistoryReloaded(chatApi, chat);
+    }
```

Comment:
There is a memory leak here: `chat` should be deleted at the end of the function.

Reply:
Done!